cdogs-sdl.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+  <id>cdogs-sdl.desktop</id>
```

Comment:
```suggestion
  <id>io.github.cxong.cdogs-sdl</id>
```

Reply:
You need a `<launchable type="desktop-id">io.github.cxong.cdogs-sdl.desktop</launchable>` too.